src/index.js

```diff
@@ -1 +1,57 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const scores = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/7oyqKQQCxsTaogFAwN0N/scores/';
+
+let scoresArray = [];
+
+const scoresList = document.getElementById('scoresList');
+
+const refreshButton = document.getElementById('refresh');
+
+function displayScores() {
+  scoresList.innerHTML = '';
+
+  fetch(scores, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((res) => res.json())
+    .then((data) => {
+      scoresArray = data.result;
+    });
+
+  scoresArray.forEach((score) => {
+    const li = document.createElement('li');
+    li.innerHTML = `<p>${score.user}: ${score.score}</p>`;
+    scoresList.append(li);
+  });
+}
+
+document.addEventListener('DOMContentLoaded', displayScores);
+refreshButton.addEventListener('click', displayScores);
```

Comment:
- [ ] <b>REQUIRED:</b> Refresh button is working only when it is clicked twice. kindly fix that.